src/unraphael/dash/align.py

```diff
@@ -5,7 +5,7 @@
 from __future__ import annotations
 
 import math
-from typing import Callable
+from typing import Callable, Tuple
```

Comment:
```suggestion
from typing import Callable, Tuple
```

Explicitly importing `Tuple` is not necessary, can just be `tuple` since Python 3.10